common/overridefileinfo.py

```diff
@@ -0,0 +1,51 @@
+"""
+Copyright (c) 2020, salesforce.com, inc.
+All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+"""
+import glob
+import os
+
+
+class OverrideFileInfo:
+
+    def __init__(self, override_file_paths):
+        self.override_file_paths = override_file_paths
```

Comment:
Generally, I would recommend passing the the file content into this class instead of the file path.  Then you can easily add a unit test for this class also (unit test just passes in strings, doesn't need to write files).

Reply:
Yeah, that can also be done. The maveninstallinfo does the same thing i.e. takes the path. I think we can create similar testcases for this too. I will try it and write the unittest cases.